### PR TITLE
infra: resource optimizations

### PR DESCRIPTION
**Pull request by idoshamun**
* Update deployments resources
* Tweak max messages (experiment)

### CODE REVIEW

**Review by omBratteng — APPROVED** [review]

File: src/common/pubsub.ts [hunk 1]
```diff
@@ -443,7 +443,7 @@ export const workerSubscribe = (
     logger: FastifyLoggerInstance,
     pubsub: PubSub,
   ) => Promise<void>,
-  maxMessages = 1,
+  maxMessages = 100,
```
Comment (omBratteng) [inline src/common/pubsub.ts:446]: Going hard 😬

File: .infra/Pulumi.prod.yaml [hunk 2]
```diff
@@ -28,7 +28,7 @@ config:
       secure: AAABALdzzBMEk2sKYDjFrWsbsdYJDOvfDfmnOmaj6lvRkds8E6SYw1RNgQ/hpCky9W8BeJXFzu2b0W6jD30KkA==
     analyticsUrl: http://analytics-api
     apiConfigFeatureKey: api-config
-    cdcWorkerMaxMessages: 1
+    cdcWorkerMaxMessages: 100
```
Comment (capJavert) [inline .infra/Pulumi.prod.yaml:31]: Will this interfere with order of CDC since now more messages will be run in parallel?

File: .infra/index.ts [hunk 3]
```diff
@@ -394,7 +394,7 @@ if (isAdhocEnv) {
       maxReplicas: 2,
       limits: { memory: '1Gi' },
       requests: {
-        cpu: '1',
+        cpu: '200m',
```
Comment (capJavert) [inline .infra/index.ts:397]: I remember digest needed more cpu, does it not use it from metrics?

Comment (omBratteng) [reply .infra/index.ts:397]: https://console.cloud.google.com/monitoring/dashboards/builder/4fd02e51-b395-43c8-97dc-852055e0ab30;duration=P14D?f.umlabel.app.App=api-personalized-digest&f.rlabel.namespace_name.Namespace=&project=devkit-prod

Comment (capJavert) [reply .infra/index.ts:397]: 👍

**Review by capJavert — APPROVED** [review]

**Pull request merged**